docs/index.md

```diff
@@ -152,6 +152,37 @@ Add to LSP settings' clients:
 }
 ```
 
+### Elm<a name="elm"></a>
+
+See instructions for installing the [elm-language-server](https://github.com/elm-tooling/elm-language-server).
+Install [Elm Language Support](https://packagecontrol.io/packages/Elm%20Language%20Support) from Package Control for syntax highlighting.
+
+Add to LSP settings' clients:
+
+```json
```

Comment:
Are all code blocks in the readme styled with `json`? I think `js` looks better overall for some reason.